apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -215,13 +227,37 @@ const state = reactive({
 });
 
 /* Components */
-const handleChangeToggle = () => {};
+const handleChangeToggle = async (value) => {
```

Comment:
nitpick: please set type of value:) And can the value be `undefined`? If so, it may cause issues with `Object.keys()`.